alpaca_trade_api/rest.py

```diff
@@ -130,9 +130,11 @@ def get_account(self):
         resp = self.get('/account')
         return Account(resp)
 
-    def list_orders(self, status=None):
-        '''Get a list of orders'''
-        params = dict()
+    def list_orders(self, params=dict(), status=None):
```

Comment:
FYI, `dict()` is initialized and reused this way, which means the second invocation will have the same dict instance with keys filled as in the first call.
https://docs.quantifiedcode.com/python-anti-patterns/correctness/mutable_default_value_as_argument.html

Reply:
fascinating, certainly explains it well and curious why I was never aware! Can change once we decide on the direction.